ui/app/AppLayouts/Browser/stores/Web3ProviderStore.qml

```diff
@@ -10,12 +10,16 @@ QtObject {
 
     property int networkId: providerModule.networkId
 
-    function revokeAllPermissions(){
-        dappPermissionsModule.revokeAllPermissions()
+    function revokeAllPermissions(address){
+        dappPermissionsModule.revokeAllPermissions(address)
     }
```

Comment:
I haven't really seen the design, but isn't there a way to remove all permissions from all accounts with a single button?

Reply:
I don t think